tests/background/hypothesis-chrome-extension-test.js

```diff
@@ -55,6 +55,7 @@ describe('HypothesisChromeExtension', function () {
   }
 
   beforeEach(function () {
+    global.chrome = { runtime: { lastError: false } };
```

Comment:
This state could potentially leak into other tests. I suggest we make sure this change is undone in an `afterEach` callback.

Reply:
Will do